src/main/java/luca/task/Deadline.java

```diff
@@ -0,0 +1,66 @@
+package luca.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Tasks that is required to be done before a specific
+ * date/time.
+ */
+public class Deadline extends Task {
+
+    /** String representation of the deadline. */
+    protected LocalDateTime by;
```

Comment:
The variable name "**by**" can be replaced with words like "time" as it is better to name the fields as **nouns**.